.babelrc.cjs

```diff
@@ -0,0 +1,5 @@
+const plugins = [
+    ['babel-plugin-direct-import', { modules: ['@mui/material', '@mui/icons-material'] }],
+];
+
+module.exports = { plugins };
```

Comment:
It doesn't really matter, but any reason this file is common JS? The rest of the project is ESM so it would be nice to stay consistent. But no worries if it is not doable, I know some libraries don't really support it.

Reply:
@Thenlie Yeah originally was not cjs but I had to change it, the Vite React plugin spat out errors otherwise.